src/Authentication/Authentication/custom/Find-MgGraphCommand.ps1

```diff
@@ -104,57 +108,129 @@ Function Find-MgGraphCommand {
         else {
             [Microsoft.Graph.PowerShell.Authentication.GraphSession]::Instance.MgCommandMetadata = GraphCommand_ReadGraphCommandMetadata
         }
+
+        function ResolveCommand {
+            param(
+                [Parameter(Mandatory = $true, Position = 0)]
+                [string]$Command
+            )
+
+            Write-Debug "Received Command: $Command"
+            [array]$ResolvedCommands = GraphCommand_ReadGraphCommandMapping | Where-Object LegacyMapping -Contains $Command | Select-Object -ExpandProperty Command
```

Comment:
We are reading `MgCommandMapping.json` file every time a user looks up a command. Let's cache the file by:
- Adding `MgLegacyCommandMapping` property to `GraphSession` at https://github.com/jasoth/msgraph-sdk-powershell/blob/features/2.0/src/Authentication/Authentication.Core/Common/GraphSession.cs#L57.
- Reading `MgCommandMapping.json` file only once and cache its contents in `[Microsoft.Graph.PowerShell.Authentication.GraphSession]::Instance.MgLegacyCommandMapping = GraphCommand_ReadLegacyGraphCommandMapping` for future reads at https://github.com/jasoth/msgraph-sdk-powershell/blob/features/2.0/src/Authentication/Authentication/custom/Find-MgGraphCommand.ps1#L108-L110.